jupyterlab_caip_optimizer/src/components/study_details.tsx

```diff
@@ -0,0 +1,292 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Paper,
+  Typography,
+  Grid,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+} from '@material-ui/core';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import * as Types from '../types';
+import { prettifyStudyName } from '../service/optimizer';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import { dateFormat, makeReadable } from '../utils';
+import { setView } from '../store/view';
+
+const StyledTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: theme.palette.action.hover,
+    color: theme.palette.common.black,
+  },
+  body: {
+    fontSize: 14,
+  },
+}))(TableCell);
+
+const StyledTableRow = withStyles(theme => ({
+  root: {
+    '&:nth-of-type(odd)': {
+      backgroundColor: theme.palette.common.white,
+    },
+  },
+}))(TableRow);
+
+const useStyles = makeStyles({
+  table: {
+    minWidth: 700,
+  },
+});
+
+interface FormattedParam {
+  name: string;
+  type: Types.ParameterType;
+  values?: number[] | string[];
+  minValue?: number | string;
+  maxValue?: number | string;
+}
+
+interface FormattedStudy {
```

Comment:
nit; I think I would just grab the actual types or declare as strings, this probably makes sense if the types will be changed somewhere down the line but that generally shouldn't happen with public APIs

Reply:
I'm a bit confused about what you're suggesting regarding grabbing the actual types. Right now, I'm grabbing all the actual types except the objective which is a new param that isn't in `Types.Study` type. Would you prefer all the params to be plain strings instead?

Reply:
Yeah I can do that, but wouldn't this way be better since if we have to change Types.Study at some point due to API change, the types would automatically align?